### ISSUE TITLE
After static theme submissions the screenshots display visual issues on old pages

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-server#7610]
STR:
1. Load AMO dev and submit a static theme.
2. Click on View Listing from Edit information to check the preview of the the theme saved as a screenshot.
3. Observe the screenshot after you click on the view classic desktop site.
4. Click on Edit information below the submitted theme and access the Listed Review Page and Unlisted Review Pages.
5. Observe the screenshots from image gallery and above.

Expected results:
Screenshots are displayed without problems such as in the new front page.

Actual results:
Another image is displayed and only after clicking it the correct screenshot becomes visible.

Notes:
This issue is reproducible when submitting a static theme by adding a file or when using the theme wizard generator and also after a new version is uploaded.
This issue is reproducible on AMO dev with FF58 (Win10).

Classic view

![classic view screenshots](https://user-images.githubusercontent.com/33448286/36430964-0a078442-165f-11e8-8503-8fcbd32331c5.gif)

Listed review page - Reviewer tools 

![listed review page](https://user-images.githubusercontent.com/33448286/36431007-220ce3fc-165f-11e8-9f34-46d8210999f3.gif)

**Comment by diox** [comment]
Marking as p2 to get some attention, but static themes are still behind a switch for now so it does not impact prod.

**Comment by eviljeff** [comment]
hah, great bug - I think it's because I didn't create "thumbnail" sized versions of the previews, so somehow another random image is being substituted.   We're only explicitly supporting Static Themes on the new frontend - where it apparently works - but this would be good to fix anyway because it's a very visible bug if someone switches to the old frontend.

**Comment by AlexandraMoga** [comment]
Actually, this can also occur on the new frontend when submitting a packed theme, in which the image `headerURL` is defined by an empty.png (i.e. used for example when submitting a multiple backgrounds theme)
See this theme as an example: https://addons-dev.allizom.org/en-US/firefox/addon/additional-bckg-standard/

![image](https://user-images.githubusercontent.com/31961530/37030293-21896ccc-2143-11e8-9aaa-c0e02cd55ca3.png)

**Comment by diox** [comment]
The weird unrelated images are probably caused by old data on dev. We typically don't see this because we overwrite existing images, but as @eviljeff points out this wasn't done for thumbnails, so that's why we see this here.

**Comment by ioanarusiczki** [comment]
I verified this issue using FF58 (Win10) for static themes submitted using upload file and the wizard features.  Also I checked the multiple backgrounds theme mentioned above by @AlexandraMoga.
I filed https://github.com/mozilla/addons-frontend/issues/4540 for what I've noticed during tests. 

Marking it verified fixed. 

![fixed for old pages](https://user-images.githubusercontent.com/33448286/37333834-f6fe7036-26b2-11e8-8fc9-6efb116ef2bf.gif)
